reload-article-from-s3.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -eu
+
+# https://s3-external-1.amazonaws.com/bucket/path/to/elife-59976-v2.xml"
+target=$1
+
+# `--type s3` how to resolve `target`. can also handle the filesystem and AWS SQS
+# `--action ingest+publish` does an ingest and the a publish in one action
+# `--force` ignore publication status
+# `--validate-only` do a 'dry run' and do not commit the transaction
+python src/adaptor.py \
+    --type s3 \
+    --target "$target" \
+    --action ingest+publish \
```

Comment:
A comment / question, I seem to remember on the `elife-bot` side of things we stopped sending `ingest+publish` in any of the ingest or publish messages. I'm not sure why, but it might have had something to do with when we added `force: true` then it didn't make any difference? I cannot remember, and I might have this wrong. I'm wondering if this would test the particular situation we observed recently. Of course, since this script is an example, it doesn't matter what the `action` value is here.

Reply:
The script is an example but I'll be tweaking it until I can pull the new xml down from s3 and pushed it into lax. I might even find the source of the bug in the process but if I don't at least it will unblock the article.

Reply:
I don't know why `ingest+publish` isn't being used, probably because it's unnecessary day to day.